tests/Any/SignerTests.cpp

```diff
@@ -216,7 +243,7 @@ TEST(Signer, NetworkNotSupported) {
     auto input = Proto::SigningInput();
     input.set_private_key("c9b0a273831931aa4a5f8d1a570d5021dda91d3319bd3819becdaabfb7b44e3b");
     input.set_transaction(transaction);
-    input.set_coin_type(TWCoinTypeBitcoinCash);
+    input.set_coin_type(TWCoinTypeZcash);
```

Comment:
Why is coinType changed here?

Reply:
Because it is supported right now, this test will fail otherwise